docs/getting-started/dev-docker-setup.md

```diff
@@ -69,7 +71,6 @@ has_toc: true
 8.5. Due to a [known bug in the current version of InvenioRDM](https://github.com/inveniosoftware/invenio-files-rest/issues/264) we can only use `setuptools` version smaller then 58.
   ```sh
   pipenv run python3.8 -m pip install 'setuptools~=57.5.0'
-  pipenv install --dev
```

Comment:
Oh. Is this redundant with `invenio-cli install --pre --development` then?

Reply:
yes exactly. both install python dependencies & update `Pipfile.lock`, so having both just slows down the process.